docker/standalone/start.sh

```
@@ -140,76 +233,116 @@ uvicorn app.main:app \
+    wait_for_http "Wework" "http://localhost:${WEWORK_PORT}" 30 "$WEWORK_PID" false
+}
+
+start_wework
+
+# ========================================
+# Step 7: Start ttyd
+# ========================================
+start_ttyd() {
+    echo "[7/8] Starting Terminal (port ${TTYD_PORT})..."
+
+    ttyd --writable -p ${TTYD_PORT} -i 0.0.0.0 bash -lc 'cd /workspace && exec bash' &
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)